src/components/common/Popup.astro

```diff
@@ -82,31 +82,29 @@ const borderSvg = `/images/popup-borders/popup-border-${size}-${color}.svg`;
 ---
 
 <div 
-  class={`popup-container relative w-full ${className}`}
+  class="popup-container w-screen"
   data-size={size}
   style={`
+    position: relative;
     aspect-ratio: ${config.aspectRatio};
     min-height: ${config.minHeight};
   `}
 >
-  <!-- Background with border SVG -->
-  <div 
-    class="absolute inset-0 bg-contain bg-center bg-no-repeat"
-    style={`background-image: url('${borderSvg}'); background-size: 100% 100%;`}
-  ></div>
   
-  <!-- Content area positioned within SVG bounds with proper clipping -->
+  <img 
+    src={borderSvg}
+    alt=""
+    class="absolute inset-0 w-full h-full object-contain"
+  />
+  
+  <!-- Content Area -->
   <div 
-    class="absolute flex items-center justify-center content-area popup-content-clip"
+    class="absolute inset-0 z-10"
     style={`
-      left: 50%;
-      top: 50%;
-      width: ${config.contentWidth};
-      height: ${config.contentHeight};
-      transform: translate(-50%, -50%);
+      padding: ${size === 'large' ? '15%' : '12%'};
     `}
   >
-    <div class="text-center w-full px-4 h-full overflow-y-auto">
+    <div class={`w-full h-full overflow-hidden ${className}`}>
```

Comment:
Please try using the [`cn`](https://github.com/isd-sgcu/fdrpkm68-frontend/blob/dev/src/lib/utils.ts) function to merge the classes together.

```suggestion
    <div class={cn("w-full h-full overflow-hidden", className)}>
```